game_patch/misc/player.cpp

```diff
@@ -207,9 +205,23 @@ void reset_local_delayed_spawn() {
 
 CodeInjection player_execute_action_respawn_req_patch{ // click to spawn
     0x004A678B,
-    [](auto& regs) {
-        if ((get_df_server_info().has_value() && get_df_server_info()->delayed_spawns) ||
-            (rf::is_server && g_alpine_server_config_active_rules.spawn_delay.enabled)) {
+    [] (auto& regs) {
+        if (!rf::is_server) {
+            constexpr float SPAWN_WAIT_TIME_SECS = 5.f;
```

Comment:
I'd like this name to be more clear, since we do have spawn delays for players as well and it could be confused as having something to do with that. Maybe just prefix it with `BOT_` would be fine?